test/lib/Elastica/Test/ClientTest.php

```diff
@@ -26,6 +26,16 @@ public function testConstruct()
     /**
      * @group functional
      */
+    public function testGetVersion()
+    {
+        $client = $this->_getClient();
+        $this->assertNotEmpty($version = $client->getVersion());
+        $this->assertSame($version, $client->getVersion());
```

Comment:
I don't think we should hardcode version of ES that is used for tests.
I'll take a look, if it's possible to expose ES_VERSION and use it here.